docs/en/observability/uptime-lightweight-monitors.asciidoc

```diff
@@ -0,0 +1,35 @@
+[[uptime-lightweight-monitors]]
+= Use lightweight monitors
+
+// What do we want to say about lightweight monitors?
```

Comment:
IMO the main difference between these and browser monitors is that there isn't a browser environment that needs to be spun-up. Therefore, these are usually quicker.